cosmos/milkyway.json

```diff
@@ -0,0 +1,114 @@
+{
+  "rpc": "https://rpc.mainnet.milkyway.zone",
+  "rest": "https://lcdmainnet.milkyway.zone",
+  "chainId": "milkyway",
+  "chainName": "MilkyWay",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/milkyway/chain.png",
+  "nodeProvider": {
+    "name": "MilkyWay Labs",
+    "email": "development@milkyway.zone"
+  },
+  "stakeCurrency": {
+    "coinDenom": "MILK",
+    "coinMinimalDenom": "umilk",
+    "coinDecimals": 6,
+    "coinGeckoId": "",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/milkyway/chain.png"
+  },
+  "walletUrl": "https://beta.milkyway.zone/",
+  "walletUrlForStaking": "",
```

Comment:
Remove these two lines. walletUrl is deprecated and walletUrlForStaking shouldn't be empty.